internal/pkg/unit/unit.go

```diff
@@ -28,11 +29,35 @@ func Run(testfiles, configfiles []string) ([]string, []string, error) {
 			return summary, details, err
 		}
 		for _, input := range inputs {
-			route, err := GetRoute(input, parsed.VirtualServices)
+			route, err := GetRoute(input, parsed.VirtualServices, true)
```

Comment:
tiny nit: prefer to have a variable as parameter to avoid browsing to the function to know what's about.
```suggestion
			checkHosts := true
			route, err := GetRoute(input, parsed.VirtualServices, checkHosts)
```